README.md

```diff
@@ -1 +1,23 @@
-# hw01
```

Comment:
You don't generally need to commit and push the HTML files. `README.html` was generated when you previewed the `README.md` in RStudio. [Check out this guide for which files are necessary to store in the repo](https://cfss.uchicago.edu/faq/homework-guidelines/#which-files-to-commit)